### PR TITLE
Update Rules

### PR DESCRIPTION
**Pull request by sagudo**
This reflects the current version of the rules. @giannic @gueldner pls review, cc @adrianlee44 @feerst

### CODE REVIEW

File: Rules.md [hunk 1]
```diff
@@ -1,58 +1,59 @@
...
 
-#### Mass Registration and Automation
+###Mass Registration and Automation
 Don’t register accounts or post content automatically, systematically, or programmatically.
 
-#### Copyright and Trademark Infringement
-Respect the copyrights and trademarks of others. Unless you’re authorized to use someone else’s copyrighted or trademarked work (either expressly or by legal exceptions and limitations like fair use), don’t do it. It’s our policy to respond to notices of alleged copyright infringement under our [Terms of Service](https://medium.com/policy/medium-terms-of-service-9db0094a1e0f) and the Digital Millennium Copyright Act.
+###Copyright and Trademark Infringement
+Respect the copyrights and trademarks of others. Unless you’re authorized to use someone else’s copyrighted or trademarked work (either expressly or by legal exceptions and limitations like fair use), don’t do it. It’s our policy to respond to notices of alleged copyright infringement under our Terms of Service and the Digital Millennium Copyright Act.
```
Comment (adrianlee44) [inline Rules.md]: The link to ToS is removed here.

Comment (sagudo) [inline Rules.md]: fixed, thanks!

Comment (sagudo) [inline Rules.md]: fixed, thanks!

**Comment by giannic** [comment]
LG.

**Comment by adrianlee44** [comment]
lgtm

**Pull request merged**